runtime/src/lib.rs

```diff
@@ -1306,13 +1306,19 @@ where
 
 parameter_types! {
     pub const DEXIdValue: DEXId = 0;
+    pub const MaxWhiteListTokens: u32 = 30;
 }
 
 parameter_types! {
     pub const FeeReferrerWeight: u32 = 10; // 10%
     pub const FeeXorBurnedWeight: u32 = 20; // 20%
     pub const FeeValBurnedWeight: u32 = 50; // 50%
     pub const FeeKusdBurnedWeight: u32 = 20; // 20%
+    pub const MinimalFeeInAsset: Balance = ((FeeReferrerWeight::get()
```

Comment:
It is minimal amount where may be calculated proportions. For swap it is used in minimal amount out, for remint_val value is zero. Also it is used for fee calculation to prevent values which cant be used in proportions. I don't if should and how should be calculated such value.

@vovac12 what do you think?

Reply:
@wer1st as I understand this value is set as minimal amount which could be used in further calculations
@ZlayaMorda Maybe just set constant value instead of such calculation and explain it in comment?

Reply:
Yeah, may set constant value, add comments where it's used

Reply:
discussed and done